src/ErrorPage.js

```diff
@@ -21,18 +21,18 @@ function ErrorPage({ resetSearch }) {
                 <SubHeading>Probiere eine neue Suche!</SubHeading>
             </Result>
             <Navigation>
-                <Button onClick={newSearch}>Neue Suche</Button>
+                <FullWidthButton onClick={newSearch}>
```

Comment:
When the HTML element name is in the beginning of a name, one could grasp it a bit faster. For example ``ButtonFullWidth``